firefox-ios/firefox-ios-tests/Tests/XCUITests/BaseTestCase.swift

```diff
@@ -54,6 +54,7 @@ class BaseTestCase: XCTestCase {
     func closeFromAppSwitcherAndRelaunch() {
         let swipeStart = app.coordinate(withNormalizedOffset: CGVector(dx: 0.5, dy: 0.999))
         let swipeEnd = app.coordinate(withNormalizedOffset: CGVector(dx: 0.5, dy: 0.001))
+        sleep(2)
```

Comment:
can't we wait for any element instead? not optimal adding sleeps.. :(

Reply:
being about closing the app from appswitcher and relaunching it i cound't find an element to wait for. Using sleep was my last choice.

Reply:
Let's try that then, thanks